Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+pipeline {
+    agent {label 'mo1833.gadi'}
+    environment {
+        DB_PATH = '/g/data/ik11/databases'
+        DB_LINK = "${DB_PATH}/cosima_master.db"
+        DB_DATE = """${sh(returnStdout: true, script: 'date +"%Y-%m-%d"')}""".trim()
+        DB      = "${DB_PATH}/daily/cosima_master_${DB_DATE}.db"
+    }
+    stages {
+        stage('Update database') {
+            steps {
+                // Check if an update of the database has already been done today. Stop with an error if that is the case.
+                script {
+                    if (fileExists("${DB}")) {
+                        error("Database has already been updated today.")
+                    }
+                }
+
+                // Start the update from the latest version of the database
+                sh '''cp ${DB_LINK} ${DB}
+                      chmod 600 ${DB}'''
+
+                // Try to update the database. Because the PBS job has a time limit, the update can fail for two different reasons:
+                //  1. the time limit was reached
+                //  2. an error occured
+                // In the first case, we don't want to mark the build as failed, because the database will still be usable.
+                script {
+                    catchError(buildResult: 'UNSTABLE', stageResult: 'UNSTABLE') {
+                       sh "qsub -v DB build_master_index"
```

Comment:
should this be
```
sh "qsub -v ${DB} build_master_index"
```
?

Reply:
No. Here we are telling PBS that there's an environment variable named `DB` that needs to be passed along when executing the job.